aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -138,6 +138,9 @@ class TORCH_API Dispatcher final {
   // Like findSchema, but also returns OperatorHandle even if there is no schema
   c10::optional<OperatorHandle> findOp(const OperatorName& operator_name);
 
+  // Returns a list of all operator names present in the operatorLookupTable_
+  const std::vector<OperatorName> getAllOpNames();
```

Comment:
don't return const objects; it defeats RVO. See e.g. https://www.bfilipek.com/2017/01/const-move-and-rvo.html